src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -139,6 +139,17 @@ class Comment extends React.Component<Props, State> {
                                 name={createdByUser.name}
                                 getUserProfileUrl={getUserProfileUrl}
                             />
+                            {!!onEdit && !!canEdit && !isPending && <InlineEdit id={id} toEdit={toEdit} />}
+                            {!!onDelete && !!canDelete && !isPending && (
+                                <InlineDelete
+                                    id={id}
+                                    permissions={permissions}
+                                    message={<FormattedMessage {...inlineDeleteMessage} />}
+                                    onDelete={onDelete}
+                                />
+                            )}
+                        </div>
```

Comment:
the important change here is closing the div here and starting another, so the text is on a new line